library/lua/5.4.4/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM --platform=linux/x86_64 debian:bookworm AS build
```

Comment:
You can remove `--platform=linux/x86_64`, this will be set automatically by KraftKit